frontend/src/pages/signup/SignUp.jsx

```diff
@@ -0,0 +1,225 @@
+      setpasswordInput(true);
+    } else {
+      setpasswordInput(false);
+    }
+  };
+
+  const ChengePasswordCheck = (e) => {
+    const value = e.target.value.replace(/(\s*)/g, '');
+
+    if (passwordInputNum === value) {
+      setpasswordCheckInput(true);
+    } else {
+      setpasswordCheckInput(false);
+    }
+  };
+
+  return (
+    <>
+      <div className={styles.signUpContainer}>
+        <form className={styles.signUpForm}>
+          <div>
+            <label className={styles.fields}>이메일*</label>
+            <input
+              className={styles.fieldsInput}
+              type="email"
+              name="email"
+              onChange={(e) => setEmail(e.target.value)}
+              value={email}
+              placeholder={placeholdersVisible[0] ? '이메일 작성' : ''}
+              onFocus={() => handleFocus(0)}
+              onBlur={(e) => handleBlur(0, e.target.value)}
+              style={{ textAlign: placeholdersVisible[0] ? 'center' : 'left' }}
+            />
+            <button className={styles.emailCheckBut} onClick={emailCheck}>
+              인증
+            </button>
+          </div>
+
+          {isCodeSent && (
+            <div>
+              <div className={styles.fields}>인증 코드 입력</div>
+              <input
+                className={styles.fieldsInput}
+                type="text"
+                placeholder={placeholdersVisible[1] ? '인증 번호 입력' : ''}
+                maxlength="6"
+                onFocus={() => handleFocus(1)}
+                onBlur={(e) => handleBlur(1, e.target.value)}
+                style={{ textAlign: placeholdersVisible[1] ? 'center' : 'left' }}
+                // onChange={(e) => setInputCode(e.target.value)}
+                onChange={ChengeverificationCode}
+                value={inputCode}
+              />
+              <div className={styles.authentication}>
+                {inputCode === verificationCode && timeLeft > 0
+                  ? '인증되었습니다!'
+                  : timeLeft > 0
+                    ? `남은 시간: ${Math.floor(timeLeft / 60)}:${('0' + (timeLeft % 60)).slice(-2)}`
+                    : '인증 코드가 만료되었습니다.'}
+              </div>
+            </div>
+          )}
+
+          <div>
+            <label className={styles.fields}>비밀번호*</label>
+            <input
+              className={styles.fieldsInput}
+              type="password"
+              name="password"
+              placeholder={placeholdersVisible[2] ? '비밀번호 작성' : ''}
+              onFocus={() => handleFocus(2)}
+              onBlur={(e) => handleBlur(2, e.target.value)}
+              style={{ textAlign: placeholdersVisible[2] ? 'center' : 'left' }}
+              onChange={ChengePassword}
+            />
+            <div className={passwordInput ? styles.visibilityVisible : styles.visibilityHidden}>
+              비밀번호는 특수부호를 포함 8자이상 작성해주세요
+            </div>
+          </div>
+
+          <div>
+            <label for="password" className={styles.fields}>
```

Comment:
```suggestion
            <label htmlFor="password" className={styles.fields}>
```

React에선 htmlFor로 사용한다고 합니다.